firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -211,209 +260,377 @@ const double MIN_PACC = 2.3;              // minumum accumulator pressure to mai
+void init_accumulator( uint8_t sensor_pin, uint8_t control_pin )
 {
-	accumulator->_sensor_pin = sensor_pin;
-	accumulator->_control_pin = control_pin;
+	accumulator._sensor_pin = sensor_pin;
+	accumulator._control_pin = control_pin;
 
-	pinMode( accumulator->control_pin, OUTPUT );
+	pinMode( accumulator._control_pin, OUTPUT );
 
-	accumulator_pump_off( accumulator );
+	turn_accumulator_pump_off( );
 }
 
 //master solenoid structure
-struct SMC_s
+struct master_cylinder_solenoid_data_s
 {
 	float _pressure1 = 0.0; // Initialize pressures to 0.0 to avoid false values
     float _pressure2 = 0.0;
-    byte _sensor1_pin = 99;
-    byte _sensor2_pin = 99;
-    byte _control_pin = 99;
-};
+    uint8_t _sensor1_pin = 99;
+    uint8_t _sensor2_pin = 99;
+    uint8_t _control_pin = 99;
+} master_cylinder_solenoid;
 
-void SMC_solenoids_open( SMC_s * const SMC )
+// *****************************************************
+// Function:    open_master_cylinder_solenoid
+// 
+// Purpose:     Open master cylinder solenoid
+// 
+// Returns:     void
+// 
+// Parameters:  None
+// 
+// *****************************************************
+void open_master_cylinder_solenoid( )
 {
-	analogWrite( SMC->_control_pin, 0 );
+	analogWrite( master_cylinder_solenoid._control_pin, 0 );
 }
 
-void SMC_solenoids_close( SMC_s * const SMC )
+// *****************************************************
+// Function:    close_master_cylinder_solenoid
+// 
+// Purpose:     Close master cylinder solenoid
+// 
+// Returns:     void
+// 
+// Parameters:  None
+// 
+// *****************************************************
+void close_master_cylinder_solenoid( )
 {
-	analogWrite( SMC->_control_pin, 255 );
+	analogWrite( master_cylinder_solenoid._control_pin, 255 );
 }
 
-void SMC_check_pedal( SMC_s * const SMC )
+// *****************************************************
+// Function:    check_driver_brake_override
+// 
+// Purpose:     Check if driver is pressing the brake
+// 
+// Returns:     void
+// 
+// Parameters:  None
+// 
+// *****************************************************
+void check_driver_brake_override( )
 {
-	SMC->_pressure1 = convert_to_voltage( analogRead( SMC->_sensor1_pin ) );
-	SMC->_pressure2 = convert_to_voltage( analogRead( SMC->_sensor2_pin ) );
+	master_cylinder_solenoid._pressure1 = convert_to_voltage( analogRead( master_cylinder_solenoid._sensor1_pin ) );
+	master_cylinder_solenoid._pressure2 = convert_to_voltage( analogRead( master_cylinder_solenoid._sensor2_pin ) );
 
 	//if current pedal pressure is greater than the limit
 	//(because of driver override by pressing the brake pedal), disable.
-	if( ( SMC->_pressure1 > PEDAL_THRESHOLD ) ||
-		( SMC->_pressure2 > PEDAL_THRESHOLD ) )
+	if( ( master_cylinder_solenoid._pressure1 > PEDAL_THRESHOLD ) ||
+		( master_cylinder_solenoid._pressure2 > PEDAL_THRESHOLD ) )
 	{
-		DEBUG_PRINT( "Brake Pedal Detected" )
+		DEBUG_PRINT( "Brake Pedal Detected" );
 		pressure_req = ZERO_PRESSURE;
 		local_override = 1;
```

Comment:
I renamed local_override in my context to driver_override, which I think makes more sense.